test/intl402/Intl/getCanonicalLocales/preferred-grandfathered.js

```diff
@@ -62,7 +62,7 @@ var regularGrandfatheredNonUTS35 = [
 
 var regularGrandfatheredUTS35 = {
   "art-lojban": "jbo",
-  "cel-gaulish": "xtg-x-cel-gaulish",
+  "cel-gaulish": "xtg",
```

Comment:
Should this also be updated in `harness/testIntl.js`?

Reply:
sure, I am not aware of such tool